File: VB/WinForms_Localizer/WinForms_Localizer/MyDashboardCoreLocalizer.vb
Imports DevExpress.DashboardCommon.Localization
Imports System

Namespace WinForms_Localizer

    Friend Class MyDashboardCoreLocalizer
        Inherits DashboardLocalizer

        Public Overrides Function GetLocalizedString(ByVal id As DashboardStringId) As String
            Return [Enum].GetName(GetType(DashboardStringId), id)
        End Function
    End Class
End Namespace

File: CS/WinForms_Localizer/WinForms_Localizer/MyDashboardCoreLocalizer.cs
using DevExpress.DashboardCommon.Localization;
using DevExpress.DashboardWin.Localization;
using DevExpress.Utils.Localization;
using System;

namespace WinForms_Localizer
{
    internal class MyDashboardCoreLocalizer : DashboardLocalizer
    {
        public override string GetLocalizedString(DashboardStringId id)
        {
            return Enum.GetName(typeof(DashboardStringId), id);
        }
    }
}